src/accelerate/utils/dataclasses.py

```diff
@@ -279,40 +287,52 @@ def __post_init__(self):
 AmaxComputeAlgorithm = Literal["max", "most_recent"]
 
 
+# FP8 training recipe kwargs
+@dataclass
+class AORecipeKwargs(KwargsHandler):
+    """
+    Use this object in your [`Accelerator`] to customize the initialization of the recipe for FP8 mixed precision
+    training with `torchao` FP8.
+
+    Args:
+        recipe_name (`str`, *optional*, default to `None`):
```

Comment:
is it ok to leave the recipe by name lookup out for now?  Technically this is a private API, we really like it internally and want to move it to public, but wanted to do a round of community feedback + design before we do that.  The default recipe (`Float8LinearConfig()`) is a great starting point!

Reply:
Sure, will tweak accordingly